docker-cloud/builds/automated-build.md

```diff
@@ -105,6 +105,17 @@ run an initial test.
     Docker Cloud on every push. Only pushes to branches that are listed as the
     source for one or more tags will trigger a build.
 
+### Environment variables for builds
+
+When you configure automated builds in Docker Cloud you can specify additional Environment Variables. The variables you add on the **Configure build settings** screen are used during the build process _only_.
+
+You can also declare environment variables in `hooks` files, however anything
```

Comment:
I understand your concern, but I want to make sure I'm introducing the concept in prose in this section before I dive in to how it works. I can definitely improve this to talk about build env vars being in Hooks too.